doc/gh-pages/overview.md

```diff
@@ -1,18 +1,20 @@
 ---
-title: About
+title: The Project
 parent: Home
-nav_order: 2
+nav_order: 1
 ---
 
+# The MNE-CPP Project
+
 ## What is MNE?
 
-MNE-CPP is part of MNE, a tool suite for neurophysiology encompassing various projects that draw from MNE-C, originally written by Matti Hämäläinen (documented [here](https://mne.tools/mne-c-manual/MNE-manual-2.7.3.pdf))
+MNE-CPP is part of [MNE Project](https://mne.tools/stable/index.html){:target="_blank" rel="noopener"}, a set of tools for neurophysiology analysis encompassing various projects that sprawn from MNE-C, originally written by Matti Hämäläinen.
```

Comment:
I'm not sure if 'sprawn' is a word. Maybe we could use 'originate', 'spawn, 'take inspiration',

Reply:
sentence might work better with 'the MNE Project' instead of just 'MNE project'

Reply:
i don't think is a word. 😄 
Done